addons/storyshots/stories/directly_required/index.js

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import { storiesOf } from '@storybook/react'; // eslint-disable-line
 import { action } from '@storybook/addon-actions'; // eslint-disable-line
 
-import Button from '@storybook/components/dist/demo/Button';
+import Button from '@storybook/react/dist/demo/Button';
```

Comment:
I'm not sure why it sometimes passes to be honest, I would have thought it would consistently fail due to: https://github.com/storybooks/storybook/pull/1303#issuecomment-310052041